synfig-studio/src/gui/widgets/widget_defaults.cpp

```diff
@@ -241,7 +246,12 @@ Widget_Defaults::Widget_Defaults():
 		_button_reset->signal_clicked().connect(sigc::mem_fun(*this,&Widget_Defaults::on_reset_color_clicked));
 		_button_reset->set_tooltip_text(_("Reset Colors to Black and White"));
 
-		Gtk::Alignment *button_reset (manage (new Gtk::Alignment(Gtk::ALIGN_CENTER, Gtk::ALIGN_CENTER, 0 ,0)));
+		Gtk::Fixed *button_reset (manage(new Gtk:: Fixed));
```

Comment:
Is it really necessary? Why don't set valign/halign of _button_reset like you did in previous commit?

Reply:
I made the requested changes now ! kindly look into it

Reply:
Please remove all these `Gtk::Fixed`. They are not meant for creating a 'fixed space'. If you want to force it with an empty widget, just use a `Gtk::Widget` ;)

Reply:
If they contain a widget though, just apply the alignment info like you did for the fill-color widget.

Reply:
Got it ! Understood ! I will start replacing Gtk : :  Fixed and Gtk : : Alignment , wherever possible !

Reply:
Instead of using `Gtk::Widget` as a replacement for `Gtk::Fixed` there's `set_margin_start/end/top/bottom()` to achieve this.

I'll requote my previous comment again here:
> We have set_margin_start() and set_margin_end() to achieve the same behavior. So instead of placing that space widget, find out where it is placed and determine if our next widget is preceding or succeeding from space widget, then use the appropriate margin property.

Reply:
BTW, there's nothing wrong with using `Gtk::Widget` as a replacement for `Gtk::Fixed`. It's still a solution though.